pkg/types/types.go

```diff
@@ -45,6 +45,7 @@ const (
 
 const (
 	GlobalMountPathPattern = "/host/var/lib/kubelet/plugins/kubernetes.io/csi/driver.longhorn.io/*/globalmount"
+	GlobalMountPathFormat  = "/host/var/lib/kubelet/plugins/kubernetes.io/csi/driver.longhorn.io/%s/globalmount"
```

Comment:
It just occurred to me that Longhorn supports customized kubelet root dir. Then is this global mount path still valid?

Reply:
Yeah you are right, let me see if we can fix it.

Reply:
Changed to use regex to build `volumeNameSHA->mountPoint` map
and then use volumeNameSHA to get the mountPoint and get the `mp.Path`
Then we don't need to know the custom kubelet directory